lib/ignore.js

```diff
@@ -28,6 +28,11 @@ function ignore(dir, options) {
       // add the xpi file name to the ignore list
       var xpiName = getID(manifest) + "*.xpi";
       defaultRules.push(xpiName);
+      // add the signed xpi file name to the ignore list
+      var addonTitle = manifest.title || manifest.name || "Untitled";
+      var signedXpiName = addonTitle.toLowerCase().replace(/ /g, "_") + "*.xpi";
+      defaultRules.push(signedXpiName);
+      console.log(signedXpiName);
```

Comment:
Yes, this was left in by accident. The packaged files are indeed listed with `-v`